lib/ansible/galaxy/collection.py

```diff
@@ -320,6 +320,7 @@ def from_name(collection, apis, requirement, force, parent=None):
                 resp = json.load(open_url(n_collection_url, validate_certs=api.validate_certs, headers=headers))
             except urllib_error.HTTPError as err:
                 if err.code == 404:
+                    display.vvv("Collection '%s' is not available from server '%s'" % (collection, api.name))
```

Comment:
It would be good to also add the URL used to this and the other message you've added below. The name isn't that useful for people who are relying on the defaults and seeing the URL as well helps them to understand what was actually used. An example is https://github.com/ansible/ansible/blob/devel/lib/ansible/galaxy/collection.py#L435

Reply:
yeah, formatting `(collection , api.name, api.api_server)` looks good to me